kumascript/macros/HTMLSidebar.ejs

```diff
@@ -43,11 +43,15 @@ var text = mdn.localStringMap({
     'HTML_Elements': 'HTML elements',
     'Global_attributes': 'Global attributes',
     'Input_types': '<code>&lt;input&gt;</code> types',
-    'Documentation': 'Documentation:',
-    'Useful_lists': 'Useful lists',
-    'Index': 'All pages index',
-    'Contribute': 'Contribute',
-    'The_MDN_project': 'The MDN project',
+    'Guides': 'Guides',
+      'Allowing_cross-origin_images_and_canvas': 'Allowing cross-origin use of images and canvas',
+      'Block-level_elements': 'Block-level elements',
+      'Date_and_time_formats': 'Date and time formats used in HTML',
+      'Inline_elements': 'Inline elements',
```

Comment:
Rather than having a raw alphabetical order, I'll put _Block-level elements_ and _Inline elements_ next to each other.
```suggestion
      'Inline_elements': 'Inline elements',
      'Date_and_time_formats': 'Date and time formats used in HTML',
```

(Same for the other languages)

Reply:
done